File: solutions/python/splitwise/split.py
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    from user import User

class Split:
    def __init__(self, user: 'User', amount: float):
        self._user = user
        self._amount = amount
    
    def get_user(self) -> 'User':
        return self._user
    
    def get_amount(self) -> float:
        return self._amount

File: solutions/java/src/splitwise/Split.java
package splitwise;

public class Split {
    protected User user;
    protected double amount;

    public Split(User user, double amount) {
        this.user = user;
        this.amount = amount;
    }

    public void setAmount(double amount) {
        this.amount = amount;
    }

    public User getUser() { return user; }
    public double getAmount() { return amount; }
}
